docs/xdebug.md

```diff
@@ -2,6 +2,12 @@
 
 `xdebug` extension is disabled by default as it causes about 20% performance hit.
 
+There two variants: you can use xdebug for debuging requests from browser or you can debug requests from console (for example Drush commands).
```

Comment:
There two ways of using xdebug, for debugging requests from browser or to debug cli scripts (for example Drush commands).